src/cityreader/cityreader.py

```diff
@@ -63,7 +81,14 @@ def cityreader(cities=[]):
 
 def cityreader_stretch(lat1, lon1, lat2, lon2, cities=[]):
   # within will hold the cities that fall within the specified region
+
   within = []
+
+  if lat1 < lat2:
+    lat1, lat2 = lat2, lat1
+    lon1, lon2 = lon2, lon1
+
+  within = [city for city in cities if lat2 < city.lat < lat1 and lon2 < city.lon < lon1]
```

Comment:
Really, really nice solution here 🏆 

And great use of the short-hand syntax for swapping variables. Cleanest solution I've seen yet!